src/test/SeleniumWebDriverTest.java

```diff
@@ -0,0 +1,30 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class SeleniumWebDriverTest {
+    @Test
+    public void testDriver() throws InterruptedException {
+        System.setProperty("webdriver.chrome.driver", "src/main/resources/chromedriver.exe");
+        WebDriver driver = new ChromeDriver();
+        driver.manage().window().maximize();
+        driver.get("https://www.selenium.dev");
+        Thread.sleep(3000);
+        driver.findElement(By.xpath("//a[@class='nav-item' and normalize-space()='Downloads']")).click();
+        String stableVersion = "3.141.59";
+        WebElement actualElement = driver.findElement(By.xpath("//a[normalize-space()='3.141.59']"));
```

Comment:
This actually works, but isn't correct. Because you just get a link with text '3.141.59', but not the exact link, which shows the latest stable version. A link with same text can be somewhere in DOM as well, so you need to select the exact link, which shows the latest stable version.